metal/classifier.py

```diff
@@ -236,6 +236,7 @@ def _train_model(self, train_data, loss_fn, dev_data=None, log_writer=None):
                     metric=val_metric,
                     verbose=False,
                     print_confusion_matrix=False,
+                    t=train_config.get("validation_task"),
```

Comment:
Rather than passing a specific kwarg in in the base class `Classifier`- which could cause issues as below- I think should just create a new dictionary in the config file, like "validation_scoring_kwargs", and then pass this whole dictionary in as the kwargs here (e.g. `**validation_scoring_kwargs`)?  Then, in the single-task end model, this can just be defined as an empty dict... and here, this can contain `t: ...`

Reply:
Got it. I'm trying to do that but I'm running into an issue. I want the argument to be called "validation_task" for the train_model method, and be called "t" for the MTEndModel score method. 

That way for example you'd call mt_em.train_model(validation_task=0..) and mt_em.score(t=0...)

Do you agree that naming makes sense? do you've an idea of how to do that if we go this route?

What I'm able to do atm is to call it either "t" or "validation_task" for both methods

Reply:
Sounds good 👍